### PR TITLE
Update Kapacitor's "Getting Started" examples to use Telegraf 0.10.0 schema

### LINKED ISSUE #185: Code in Kapacitor's getting started guide doens't work with Telegraf >= 0.10.0
I tried to follow [Kapactior's getting started guide](https://docs.influxdata.com/kapacitor/v0.10/introduction/getting_started/), but I could not get it working. 

The docs mention explicitly that you must use Telegraf >= 0.1.9, but the provides sample code will fail when you use Telegraf >= 0.10.0.  Starting from this release Telegraf uses only 1 measurement for all values per plugin.

### PR DESCRIPTION
**Pull request by rossmcdonald**
Fixes #185

### CODE REVIEW

**Comment by rkuchan** [comment]
The query under "There should be some cpu metrics in a database called `kapacitor_example`. Confirm this with this query:" should be: 

```
curl -G 'http://localhost:8086/query?db=kapacitor_example' --data-urlencode 'q=SELECT count(usage_idle) FROM cpu'
```

**Comment by rossmcdonald** [comment]
@rkuchan Thanks, I added a fix for the curl command.

I also removed the misleading section about the debug output. The wording may need to be fixed now, but hopefully it shouldn't cause any more confusion.

File: content/kapacitor/v0.10/introduction/getting_started.md [hunk 1]
```diff
@@ -69,20 +62,18 @@ The following is a simple Telegraf configuration file that will send just cpu me
...
     database = "kapacitor_example"
     user_agent = "telegraf"
 
 # Collect metrics about cpu usage
-[cpu]
+[[inputs.cpu]]
     percpu = false
     totalcpu = true
-    drop = ["cpu_time"]
+    drop = ["time_*"]
```
Comment (nathanielc) [inline content/kapacitor/v0.10/introduction/getting_started.md]: These changes look good, do you want me to add then to #194?

Comment (rossmcdonald) [inline content/kapacitor/v0.10/introduction/getting_started.md]: @nathanielc Sure.

**Comment by rossmcdonald** [comment]
Closing in favor of #194

**Pull request closed without merging**